.idea/.gitignore

```diff
@@ -0,0 +1,3 @@
+# Default ignored files
```

Comment:
add below folder also in the same file to ignore:
.gradle/ 
.idea/
build/

Reply:
Done,